AsyncDisplayKit/ASCellNode.mm

```diff
@@ -170,14 +170,40 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
 
 - (void)didRelayoutFromOldSize:(CGSize)oldSize toNewSize:(CGSize)newSize
 {
-  if (_layoutDelegate != nil) {
+  if (_interactionDelegate != nil) {
     ASPerformBlockOnMainThread(^{
       BOOL sizeChanged = !CGSizeEqualToSize(oldSize, newSize);
-      [_layoutDelegate nodeDidRelayout:self sizeChanged:sizeChanged];
+      [_interactionDelegate nodeDidRelayout:self sizeChanged:sizeChanged];
     });
   }
 }
 
+- (void)setSelected:(BOOL)selected
+{
+  if (_selected != selected) {
+    _selected = selected;
+    [_interactionDelegate nodeSelectedStateDidChange:self];
+  }
+}
+
+- (void)setHighlighted:(BOOL)highlighted
+{
+  if (_highlighted != highlighted) {
+      _highlighted = highlighted;
+      [_interactionDelegate nodeHighlightedStateDidChange:self];
+    }
```

Comment:
It looks like this method has too much indentation after the if.  The one above is fine.